.github/stale.yml

```diff
@@ -30,14 +30,16 @@ issues:
   # TODO: Consider increasing the limitPerRun once we are satisfied with the bot's performance
   limitPerRun: 1
   daysUntilStale: 30
-  daysUntilClose: 7
+  daysUntilClose: 30
   markComment: >
     This issue has been automatically marked as stale because it has not had activity in the
-    last 30 days. It will be closed in the next 7 days unless it is tagged "help wanted" or other activity
+    last 30 days. It will be closed in the next 30 days unless it is tagged "help wanted" or other activity
     occurs. Thank you for your contributions.
   closeComment: >
     This issue has been automatically closed because it has not had activity in the
-    last 37 days. If this issue is still valid, please ping a maintainer and ask them to label it as "help wanted".
+    last 60 days. If this issue is still valid, please ping a maintainer and ask them to label it as "help wanted".
     Thank you for your contributions.
   exemptLabels:
     - help wanted
+    - kind/customer_issue
```

Comment:
It looks like we use a space instead of a underscore for `customer issue` https://github.com/istio/istio/labels/kind%2Fcustomer%20issue